src/test/java/marquez/api/resources/NamespaceResourceTest.java

```diff
@@ -102,6 +102,19 @@ public void testGetNamespaceRequestErrorHandling() throws Throwable {
         resources.target("/api/v1/namespaces/" + NAMESPACE_NAME).request().get().getStatus());
   }
 
+  @Test
+  public void testListNamespaceWithNoResults() throws MarquezServiceException {
+    final List<Namespace> existingCoreModelNamespaces = Collections.emptyList();
+    NamespaceService namespaceService = mock(NamespaceService.class);
+    NamespaceResource namespaceResource = new NamespaceResource(namespaceService);
+    when(namespaceService.getAll()).thenReturn(existingCoreModelNamespaces);
+
+    Response res = namespaceResource.list();
+    NamespacesResponse responseBody = (NamespacesResponse) res.getEntity();
+
+    assertThat(responseBody.getNamespaces()).isEmpty();
+  }
+
```

Comment:
Minor: We'll need to revisit the test suite for `NamespaceResource`, so would have been nice to see this test added then vs this PR on `DatasetDao`

Reply:
Yea, this was something of a non-sequitur. It got bundled into the issue b/c I think it somehow came up in the code review. But yea, better if it was part of a different PR.